app/controllers/marks_graders_controller.rb

```diff
@@ -57,7 +57,8 @@ def download_grader_students_mapping
           entry.grade_entry_form_id == grade_entry_form.id
         end
         unless grade_entry_student.nil?
-          grade_entry_student.tas.each { |ta| student_array.push(ta.user_name) }
+          grade_entry_student.tas.order(:user_name).each {
+              |ta| student_array.push(ta.user_name) }
```

Comment:
Expression at 61, 53 should be on its own line.<br>Block argument expression is not on the same line as the block start.